.travis.yml

```diff
@@ -1,11 +1,18 @@
+sudo: required
 language: node_js
 node_js:
-  - "8"
+- 8.9.4
```

Comment:
this is bc docker?

Reply:
Unfortunately not. 

It's travis being overzealous and me giving up on fighting. 

I typically try to make my changes as directed as possible but travis changes .travis.yml back every time you run the encrypt-key command so I finally gave up. :upside_down_face:

Reply:
roger that.  we should put something in the README about that (in the docker branch 👍 )